lib/Header/Header.js

```diff
@@ -616,6 +616,7 @@ module.exports = kind(
 	* @private
 	*/
 	typeChanged: function () {
+		this.addRemoveClass('moon-large-header', this.get('type') == 'large');
```

Comment:
Ah didn't realize we're calling that at create time, let's leave it as-is rather than changing that to `this.typeChanged(null, this.type);`. :+1: